src/content/en/2021/media.md

```diff
@@ -1,20 +1,900 @@
+      </tr>
+      <tr>
+        <td>`[]`</td>
+      </tr>
+      <tr>
+        <td>`["video/mp4; codecs="avc1.42E01E, mp4a.40.2","video/webm; codecs="vp8, vorbis"`</td>
+      </tr>
+      <tr>
+        <td>`["video/mp4;","video/webm;"]`</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="The most common ordered pairs of `type` values, when there are two `source` elements within a `video` element.", sheets_gid="1549760253", sql_file="video_source_formats_mobile.sql") }}</figcaption>
+</figure>
+
+{# TODO I'm not sure this is such an interesting table. Consider adding some stats to make it more interesting? % of usage on mobile and desktop? #}
+
+In six of the top 10 examples, the MP4 is listed as the first source. <a hreflang="en" href="https://caniuse.com/mpeg4">MP4 support on the web is at 98.4%</a>, and the browsers that do not support MP4 generally do not support the `<video>` tag at all. This implies that these sites do not need two sources, and could save some storage on their web servers by removing their WebM or Ogg video sources. (Or, they could reverse the order of the videos, and browsers that support WebM will download the WebM).
+
+{# TODO - second time you meantioned and links to the 9* MP4 stat, but this time it's 98.4% instead fo 98.1%. Do we need more references and do both need to be links? If so can we at least be consistent. #}
+
+The same trend holds for `<video>` elements with three sources—eight of the top 10 examples begin with MP4.
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Format</th>
+      </tr>
+    </thead>
+    </tbody>
+      <tr>
+        <td>`["video/mp4","video/webm","video/ogg"]`</td>
+      </tr>
+      <tr>
+        <td>`["video/mp4; codecs=avc1","video/mp4; codecs=avc1","video/mp4; codecs=avc1"]`</td>
+      </tr>
+      <tr>
+        <td>`["video/webm","video/mp4","video/ogg"]`</td>
+      </tr>
+      <tr>
+        <td>`["video/mp4; codecs=avc1"]`</td>
+      </tr>
+      <tr>
+        <td>`["video/mp4","video/ogg","video/webm"]`</td>
+      </tr>
+      <tr>
+        <td>`["video/mp4;","video/ogg; codecs="theora, vorbis","video/webm; codecs="vp8, vorbis"]`</td>
+      </tr>
+      <tr>
+        <td>`["video/mp4; codecs=hevc","video/webm","video/mp4"]`</td>
+      </tr>
+      <tr>
+        <td>`["video/mp4"]`</td>
+      </tr>
+      <tr>
+        <td>`["video/ogg; codecs="theora, vorbis","video/webm","video/mp4"]`</td>
+      </tr>
+      <tr>
+        <td>`["video/mp4","video/webm","video/ogv"]`</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="The most common ordered triplets of `type` values, when there are three `source` elements within a `video` element.", sheets_gid="1549760253", sql_file="video_source_formats_mobile.sql") }}</figcaption>
+</figure>
+
+{# TODO I'm not sure this is such an interesting table. Consider adding some stats to make it more interesting? % of usage on mobile and desktop? #}
```

Comment:
@dougsillars same questions here as previous - should this be a figure table? If so should we add some stats?

Or should it be a bullet list instead?

Reply:
Added the stats in https://github.com/HTTPArchive/almanac.httparchive.org/pull/2705/commits/0b9689ecd8d3a77cfc6de9d8d4de3127933a90c1